package-lock.json

```diff
@@ -22665,7 +22665,7 @@
         },
         "got": {
           "version": "9.6.0",
-          "resolved": "https://registry.npmjs.org/got/-/got-9.6.0.tgz",
+          "resolved": "http://registry.npmjs.org/got/-/got-9.6.0.tgz",
```

Comment:
This is now http ?

Reply:
yeah I've ran into this before - I just ran `npm i`, had similar discussion with Stephen D here: https://github.com/alphagov/pay-frontend/pull/1828#discussion_r501548889